CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 Unreleased
 ------
+
+Version 1.7.0
```

Comment:
This release also removes the shopifolk-beta flag, but I haven't listed it here since it's internal-facing.